src/main/java/edu/hm/hafner/analysis/parser/MsBuildParser.java

```diff
@@ -133,7 +133,7 @@ private Severity determinePriority(final Matcher matcher) {
         if (isOfType(matcher, "warning")) {
             return Severity.WARNING_NORMAL;
         }
-        return Severity.WARNING_HIGH;
+        return Severity.ERROR;
```

Comment:
Maybe it would make more sense to remove the method at all and use [Severity#guessFromString](https://github.com/jenkinsci/analysis-model/blob/master/src/main/java/edu/hm/hafner/analysis/Severity.java#L95).

I think adding a `note` as `LOW` would make sense in that method as well.

Reply:
thanks for the inputs, I will rework it and push the branch again (rebased).